hydrus/hydraspec/doc_writer.py

```diff
@@ -377,20 +377,7 @@ def generate(self):
                 "label": self.name,
                 "description": "The %s collection" % (self.name,),
                 "domain": "vocab:EntryPoint",
-                "range": "vocab:%sCollection" % (self.name,),
-                "supportedOperation": [
-                    {
-                        "@id": "_:%s_collection_retrieve" % (self.name.lower(),),
-                        "@type": "hydra:Operation",
-                        "method": "GET",
-                        "label": "Retrieves all %s entities" % (self.name,),
-                        "description": None,
-                        "expects": None,
-                        "returns": "vocab:%sCollection" % (self.name,),
-                        "statusCodes": [
-                        ]
-                    }
```

Comment:
As discussed in the mailing list, the use of Collection or Class for creating new object is subjective. But the GET operation is inherently supported. APIs may use api/class/ to create objects instead of api/classCollection.
But it's fine, we'll have to modify the operations in the Collection then